woocommerce/woocommerce
# Add/jetpack jwt library

### Changes proposed in this Pull Request:

Import and utilize the requestJWT function provided by the [@automattic/jetpack-ai-client library](https://github.com/Automattic/jetpack-ai-client/tree/trunk).

Closes #40299.

### How to test the changes in this Pull Request:

The `useBackgroundRemoval` hook utilizes `requestJwt()` now behind the scenes, so, that's a good place to run tests and check that things are working properly.

1. Run `pnpm run test` and observe that the tests pass (they have been refactored now that the `useBackgroundRemoval` hook utilizes `requestJwt()` now behind the scenes ).
2. Test all of the things with Woo AI since the calls to API services require a JWT.



### Changelog entry




<details>

#### Significance




#### Type




#### Message Woo AI - Add the @automattic/jetpack-ai-client library for JWT creation.

#### Comment 

</details>
